fiftyone/core/dataset.py

```diff
@@ -6137,6 +6176,7 @@ def _create_dataset(
     dataset_doc = foo.DatasetDocument(
         id=_id,
         name=name,
+        url_name=url_name,
```

Comment:
since this is not in prod yet, for consistency with teams, should we use `slug`?

cc @j053y

Reply:
I will try raising this again on `saved-views` branch